tspp.java

```diff
@@ -55,23 +55,23 @@ void display()
 
 void dfs_1()
 {
-Stack <Integer> s=new Stack<Integer> ();
+@LTLengthOf("mat") Stack <Integer> s=new Stack<Integer> ();
```

Comment:
If you want to express that the stack only contains integers of some kind (`@LTLengthOf("mat")`), then you should write `Stack <@LTLengthOf("mat") Integer>`. Please make sure you understand the difference between these two.

Reply:
Thanks.I didn't knew about this.!